src/System.Drawing.Primitives/pkg/System.Drawing.Primitives.pkgproj

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Drawing.Primitives.csproj">
+    </ProjectReference>
```

Comment:
:+1: add a project reference to src/System.Drawing.Primitives.builds and it should do the trick.  If you build the pkgproj it will produce a nupkg in your output directory.